ai/embedding.py

```
@@ -41,26 +58,63 @@ async def generate_hybrid_embeddings(file_id=None):
+        _pincone_index.delete(
+            namespace=file_id,
+            delete_all=True
+        )
+    except PineconeApiException as e:
+
+        return {
+            "success": False,
+            "message": str(e)
+        }
+
+    except Exception as e:
```

Ruff (0.15.12)

[warning] Do not catch blind exception: `Exception`

(BLE001)